share-web/viewer/src/style.css

```
@@ -1,23 +1,380 @@
+  --code: #94a3b8;
+  --link: #7dd3fc;
+  --quote: #34d399;
+  --success: #4ade80;
+  --user: #22d3ee;
+  --output: #60a5fa;
+  --canvas: #020617;
+  --panel: #0f172a;
+  --border: #334155;
+  --text: #e2e8f0;
+  --shadow: 0 18px 50px rgba(2, 6, 23, 0.55);
+  font-family: ui-sans-serif, system-ui, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)